admin/pages/licenses.php

```diff
@@ -10,95 +10,108 @@
+			'desc'      => sprintf( __( 'The premium version of %1$s with more features & support.', 'wordpress-seo' ), 'Yoast SEO' ),
+			'image'     => plugins_url( 'images/extensions-premium-ribbon.png', WPSEO_FILE ),
+			'benefits'  => array(),
+		)
+	)
 );
 
-if ( class_exists( 'WPSEO_Premium' ) ) {
-	$extensions['seo-premium']->installed = true;
-}
-if ( class_exists( 'wpseo_Video_Sitemap' ) ) {
-	$extensions['video-seo']->installed = true;
-}
-if ( class_exists( 'WPSEO_News' ) ) {
-	$extensions['news-seo']->installed = true;
-}
-if ( defined( 'WPSEO_LOCAL_VERSION' ) ) {
-	$extensions['local-seo']->installed = true;
-}
-if ( ! class_exists( 'Woocommerce' ) ) {
-	unset( $extensions['woocommerce-seo'] );
-}
-elseif ( class_exists( 'Yoast_WooCommerce_SEO' ) ) {
-	$extensions['woocommerce-seo']->installed = true;
-}
+$extensions->add(
+	'wpseo-video',
+	new WPSEO_Extension(
+		array(
+			'buyUrl'    => WPSEO_Shortlinker::get( 'https://yoa.st/zx/' ),
+			'infoUrl'   => WPSEO_Shortlinker::get( 'https://yoa.st/zw/' ),
+			'title'     => 'Video SEO',
+			'desc'      => __( 'Optimize your videos to show them off in search results and get more clicks!', 'wordpress-seo' ),
+			'image'     => plugins_url( 'images/extensions-video.png', WPSEO_FILE ),
+			'benefits'  => array(
+				__( 'Show your videos in Google Videos', 'wordpress-seo' ),
+				__( 'Enhance the experience of sharing posts with videos', 'wordpress-seo' ),
+				__( 'Make videos responsive through enabling fitvids.js', 'wordpress-seo' ),
+			),
+		)
+	)
+);
+
+$extensions->add(
+	'wpseo-news',
+	new WPSEO_Extension(
+		array(
+			'buyUrl'    => WPSEO_Shortlinker::get( 'https://yoa.st/zv/' ),
+			'infoUrl'   => WPSEO_Shortlinker::get( 'https://yoa.st/zu/' ),
+			'title'     => 'News SEO',
+			'desc'      => __( 'Are you in Google News? Increase your traffic from Google News by optimizing for it!', 'wordpress-seo' ),
+			'image'     => plugins_url( 'images/extensions-news.png', WPSEO_FILE ),
+			'benefits'  => array(
+				__( 'Optimize your site for Google News', 'wordpress-seo' ),
+				__( 'Immediately pings Google on the publication of a new post', 'wordpress-seo' ),
+				__( 'Creates XML News Sitemaps', 'wordpress-seo' ),
+			),
+		)
+	)
+);
 
+$extensions->add(
+	'wpseo-local',
+	new WPSEO_Extension(
+		array(
+			'buyUrl'    => WPSEO_Shortlinker::get( 'https://yoa.st/zt' ),
+			'infoUrl'   => WPSEO_Shortlinker::get( 'https://yoa.st/zs' ),
+			'title'     => 'Local SEO',
+			'desc'      => __( 'Rank better locally and in Google Maps, without breaking a sweat!', 'wordpress-seo' ),
+			'image'     => plugins_url( 'images/extensions-local.png', WPSEO_FILE ),
+			'benefits'  => array(
+				__( 'Get found by potential clients', 'wordpress-seo' ),
+				__( 'Easily insert Google Maps, a store locator, opening hours and more', 'wordpress-seo' ),
+				__( 'Improve the usability of your contact page', 'wordpress-seo' ),
+			),
+		)
+	)
+);
+
+// Only add WooCommerce when the plugin is active.
```

Comment:
`when the WooCommerce plugin is active` <-- Still not very clear. I am trying to find a way to communicate this is not about our plugins